main/src/main/scala/sbt/internal/LibraryManagement.scala

```diff
@@ -10,18 +10,19 @@ package internal
 
 import java.io.File
 import java.util.concurrent.Callable
+
 import sbt.internal.librarymanagement._
 import sbt.librarymanagement._
 import sbt.librarymanagement.syntax._
-import sbt.util.{ CacheStore, CacheStoreFactory, Logger, Tracked, Level }
+import sbt.util.{ CacheStore, CacheStoreFactory, Level, Logger, Tracked }
 import sbt.io.IO
 import sbt.io.syntax._
 import sbt.Project.richInitializeTask
 import sjsonnew.JsonFormat
 // This is to turn off "`state` is inside the if expression of a regular task." etc
-import sbt.dsl.LinterLevel.Ignore
```

Comment:
Could you also delete the comment right above it plz?

Reply:
done